Tests/UnitTests/Networking/HTTPClientTests.swift

```diff
@@ -933,16 +947,17 @@ class HTTPClientTests: TestCase {
             return response
         }
 
-        let obtainedError: Atomic<NetworkError?> = nil
-        self.client.perform(.init(method: .get, path: path)) { (result: HTTPResponse<Data>.Result) in
-            obtainedError.value = result.error
+        let obtainedError: NetworkError? = waitUntilValue { completion in
```

Comment:
Does this need to be a `Atomic`?

Reply:
Not with `waitUntilValue` anymore since it simply returns the value :) it doesn’t múrate anything.